### DISCUSSION TITLE
/posts/2023/02/04Vue3%E7%9B%B8%E5%85%B3%E5%8E%9F%E7%90%86%E6%A2%B3%E7%90%86

### DISCUSSION THREAD

**Discussion by giscus** [Justin3go/justin3go.com#113] [Announcements]
# /posts/2023/02/04Vue3%E7%9B%B8%E5%85%B3%E5%8E%9F%E7%90%86%E6%A2%B3%E7%90%86

坚持深耕技术领域的 T 型前端程序员, 关注独立开发与 AI，喜欢 Vuejs、Nestjs, 还会点 Python、搜索引擎、NLP、Web3、后端

https://justin3go.com/posts/2023/02/04Vue3%E7%9B%B8%E5%85%B3%E5%8E%9F%E7%90%86%E6%A2%B3%E7%90%86

**Comment by Justin3go** [comment]
这里处理乱序序列中，其实节点【0】也不需要移动
